Adabot_Operation_Game/adabot_operation_game.py

```diff
@@ -0,0 +1,42 @@
+# Adabot Operation Game
+#  CPX, alligator clips, copper tape, tweezers, surgery, and fun!
+
+import board
+import touchio
+from adafruit_circuitplayground.express import cpx
+# import time  # uncomment if testing raw read values
+
+cap_pins = (board.A1, board.A2, board.A3, board.A4, board.A5,
+            board.A6, board.A7)
+touch_pads = []
+for i in range(7):
+    touch_pads.append(touchio.TouchIn(cap_pins[i]))
+for touch_pad in touch_pads:
+    touch_pad.threshold = 3500  # adjust value to fine-tune touch threshold
+
+MAGENTA = (10, 0, 10)
```

Comment:
Is there a reason you chose to go with lower numbers in the tuples instead of full values and a line to lower the brightness overall? This is a perfectly excellent way to do it, however, I wonder if making the tuple values higher and allowing for brightness control might be good for people who build this project in case they want the lights to be brighter. Easier to update a brightness value on the fly than a series of tuples. But this is entirely a matter of opinion, and does not need to be changed!

Reply:
great point! I'm going to leave this one as is in this case, so people can look at the color blending/brightness equation in a different way.

Reply:
Sounds good!